phys2bids/heuristics/heur_euskalibur.py

```diff
@@ -0,0 +1,93 @@
+import fnmatch
```

Comment:
I assume this one is an example?

Reply:
Yep, why not.
it's the one used for a dataset that will be shared at a certain point.